code/controllers/subsystem/air.dm

```diff
@@ -444,9 +444,9 @@ SUBSYSTEM_DEF(air)
 				var/static/pipenetwarnings = 10
 				if(pipenetwarnings > 0)
 					log_mapping("build_pipeline(): [item.type] added to a pipenet while still having one. (pipes leading to the same spot stacking in one turf) around [AREACOORD(item)].")
-					pipenetwarnings--
 				if(pipenetwarnings == 0)
 					log_mapping("build_pipeline(): further messages about pipenets will be suppressed")
+				pipenetwarnings--
```

Comment:
This doesn't look like it fixes it at all. The proper way to fix this is to move the `if (pipenetwarnings == 0)` check underneath the `pipenetwarnings--`, which should stay inside the if block, and then indent it one up so that its only called when pipenetwarnings WASN'T zero, and was MADE zero.

Reply:
Oh wait I see why this works, it's because it'll go in the negative...but it's silly. My way is better IMO

Reply:
Not a bad point. I did it this way because this is how it's done in a separate file (which makes it really confusing that the bug exists??). I've updated both to follow your suggestion instead.